test/membrane_element_flac_parser/integration_test.exs

```diff
@@ -0,0 +1,44 @@
+defmodule ParsingPipeline do
+  @moduledoc false
+
+  alias Membrane.Testing.Pipeline
+
+  def make_pipeline(in_path, out_path, pid \\ self()) do
+    Pipeline.start_link(%Pipeline.Options{
+      elements: [
+        file_src: %Membrane.Element.File.Source{location: in_path},
+        parser: Membrane.Element.FLACParser,
+        sink: %Membrane.Element.File.Sink{location: out_path}
+      ],
+      monitored_callbacks: [:handle_notification],
+      test_process: pid
+    })
+  end
+end
+
+defmodule Membrane.Element.FLACParser.IntegrationTest do
+  use ExUnit.Case
+  import Membrane.Testing.Pipeline.Assertions
+  alias Membrane.Pipeline
+
+  def fixture_paths(filename) do
```

Comment:
this seems not only to return paths but do some side effects also. rename properly

Reply:
and should be private